lib/flight_job/help_formatter.rb

```diff
@@ -0,0 +1,225 @@
+#==============================================================================
+# Copyright (C) 2020-present Alces Flight Ltd.
+#
+# This file is part of Flight Job.
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which is available at
+# <https://www.eclipse.org/legal/epl-2.0>, or alternative license
+# terms made available by Alces Flight Ltd - please direct inquiries
+# about licensing to licensing@alces-flight.com.
+#
+# Flight Job is distributed in the hope that it will be useful, but
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR
+# IMPLIED INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS
+# OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY OR FITNESS FOR A
+# PARTICULAR PURPOSE. See the Eclipse Public License 2.0 for more
+# details.
+#
+# You should have received a copy of the Eclipse Public License 2.0
+# along with Flight Job. If not, see:
+#
+#  https://opensource.org/licenses/EPL-2.0
+#
+# For more information on Flight Job, please visit:
+# https://github.com/openflighthpc/flight-job
+#==============================================================================
+# This help formatter has been modified from commander-openflighthpc. All
+# changes are made available under the EPL-2.0 mentioned above. The original
+# work is available under the following licenses.
+#
+# The following license applies to changes made subsequent to the point
+# this version was forked from the original upstream version available
+# at <https://github.com/commander-rb/commander>.
+#
+# The MIT License (MIT)
+#
+# Copyright (c) 2017-present Alces Flight Ltd <licensing@alces-flight.com>
```

Comment:
Why do we have two licences for Alces Flight Ltd?

Reply:
Because I pulled the `help_formatter.rb` from `commander-openflighthpc` which is licensed under MIT to be consistent with the original `commander` gem.

We *maybe* able to change the alces component of the license to `EPL`? However, I haven't put much thought into the legality of doing so.

Eventually this code will likely be backported to `commander`, so I didn't feel it was worth dealing with licensing details.